src/DetailsView/components/adhoc-tab-stops-test-view.tsx

```diff
@@ -43,6 +44,7 @@ import * as Markup from '../../assessments/markup';
 
 export type AdhocTabStopsTestViewDeps = {
     detailsViewActionMessageCreator: DetailsViewActionMessageCreator;
+    userConfigMessageCreator: UserConfigMessageCreator;
```

Comment:
Would be better to cvreate/add this to the deps for AutoDetectedFailuresDialog rather than being passed in as a prop.